src/main/java/de/tum/in/www1/artemis/repository/iris/IrisChatSessionRepository.java

```diff
@@ -14,15 +16,41 @@
  */
 public interface IrisChatSessionRepository extends JpaRepository<IrisChatSession, Long> {
 
-    Optional<IrisChatSession> findByExerciseIdAndUserId(Long exerciseId, Long userId);
+    @Query("""
+                SELECT s
+                FROM IrisChatSession s
+                WHERE s.exercise.id = :exerciseId
+                AND s.user.id = :userId
+            """)
+    Optional<List<IrisChatSession>> findByExerciseIdAndUserId(Long exerciseId, Long userId);
```

Comment:
Do we actually still need this? I think we could just always use the ordered query, so that we just have one `findByExerciseIdAndUserId` method that does what `findNewestByExerciseIdAndUserIdHelper` does